File: tweeter-z-frontend/tweeter/src/app/model/comment-dto.ts
export class CommentDTO {
    commentId!: number;
    tweetId!: number;
    userId!: number;
    userHandle!: string;
    
    content!: string;
    createdAt!: Date;
    updatedAt!: Date;
  
    constructor(init?: Partial<CommentDTO>) {
      Object.assign(this, init);
    }
  }
  

File: tweeter-user-timeline-service/src/main/java/com/tweetapp/model/CommentDTO.java
package com.tweetapp.model;


import lombok.Data;

import java.time.LocalDateTime;
import java.util.Date;

@Data
public class CommentDTO {
    private Long commentId;
    private Long tweetId;
    private Long userId;
    private String content;
    private Date createdAt;
    private Date updatedAt;

    private String userHandle;

}
